src/enc/backward_references/hash_to_binary_tree.rs

```diff
@@ -170,7 +170,8 @@ where
         num_nodes = input_size;
     }
     let window_mask = (1 << params.lgwin) - 1;
-    let invalid_pos = 0u32.wrapping_sub(window_mask) as u32;
+    // FIXME: BUG? Should this be u32::MAX instead?
```

Comment:
https://github.com/google/brotli/blob/master/c/enc/hash_to_binary_tree_inc.h#L64 Seems to be same here

Reply:
Thx, i think we should just add these TODOs to the code for now, so that we don't loose track of them, but they shouldn't prevent from merging